### DISCUSSION TITLE
How to get CVC5 version information via Java API?

### DISCUSSION THREAD

**Discussion by kfriedberger** [cvc5/cvc5#9063] [Q&A] [answered]
Hi.
Is there a way to get the exact version string like "1.0.1" from a compiled CVC5 binary/library via the Java API?
Best,
Karlheinz

**Comment by aniemetz** [comment, marked as answer]
Right now, this is not possible. We can add that in the future, will keep you posted.

**Comment by aniemetz** [comment]
https://github.com/cvc5/cvc5/pull/9093
